docs/azure/sdk/logging.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Logging with the Azure SDK for .NET
+description: Learn how to enable logging with the Azure SDK for .NET client libraries
+ms.date: 03/20/2020
+ms.custom: azure-sdk-dotnet
+ms.author: casoper
+author: camsoper
+---
+
+# Logging with the Azure SDK for .NET
+
+The [Azure SDK](https://azure.microsoft.com/en-us/downloads/) for .NET client libraries includes the ability to log client library operations. This allows you to monitor I/O requests and responses that client libraries are making to Azure services. Typically, the logs are used to debug or diagnose communication issues. This article describes three approaches to enable logging with the Azure SDK for .NET:
+
+- Log to the console window
+- Log to .NET diagnostics traces
+- Configure custom logging
+
+> [!NOTE]
+> This article applies to client libraries that use the most recent versions of the Azure SDK for .NET. To see if a library is supported, refer to the list of [Azure SDK latest releases](https://azure.github.io/azure-sdk/releases/latest/js.html). If your application is using an older version of the Azure SDK client libraries, refer to specific instructions in the applicable service documentation.
+
+## Log information
+
+The SDK logs the following information, sanitizing parameter query and header values to remove personal data.
+
+HTTP request log entry:
+
+- Unique ID
+- HTTP method
+- URI
+- Outgoing request headers
+
+HTTP response log entry:
+
+- Duration of I/O operation (time elapsed)
+- Request ID
+- HTTP status code
+- HTTP reason phrase
+- Response headers
+- Error information, when applicable
+
+For request and response content:
+
+- Content stream as text or bytes depending on the Content-Type header.
+     > [!NOTE}
+     > Content logging is disabled by default. To enable it, set `Diagnostics.IsLoggingContentEnabled` to `true` in `ClientOptions`.
+
+Event logs are output usually at one of these three levels:
```

Comment:
Pavel suggested adding sample output. Thoughts?

Reply:
I'm not opposed if you have some. I wouldn't make it a huge priority, though.

Reply:
I don't have any now...